.circleci/config.yml

```diff
@@ -59,11 +61,20 @@ jobs:
       - attach_workspace:
           at: ~/project
       - run:
-          command: 'npm run check:test:ci'
+          name: 'Run tests'
+          command: |
+            # download codeclimate test reporter as a static binary
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
Could you find a docker image with this included? The caching will speed this up instead of downloading it every time

Reply:
There's about a 4-5 second difference between test runs that do this and ones that don't. I don't think it's worth creating a docker image (and then keeping it up to date with the latest node versions).